_rules/iframe-identical-name-equivalent-purpose-4b1c6c.md

```diff
@@ -209,7 +209,7 @@ Multiple `iframe` elements in the document having different `aria-labelledby` de
 
 #### Inapplicable Example 5
 
-`iframe` having the same `title` within a given document tree, but one of them is not [included in the accessibility tree][].
+Both `iframe` elements have the same [accessible name][] (given by the `title` attribute) within a the same [document tree][], but one of them is not [included in the accessibility tree][]. Therefore, there is no set of two or more `iframe` elements with the same [accessible name][] and [included in the accessibility tree][] and the rule is inapplicable.
```

Comment:
same [accessible name][] and [included in the accessibility tree][] > same [accessible name][] [included in the accessibility tree][]

Reply:
It's the `iframe`, not the accessible name, that need to be included in the accessibility tree.
I tried to improve the formulation.